src/components/Registration/RegistrationForm.jsx

```diff
@@ -59,52 +59,77 @@ export const RegistrationForm = () => {
+            <ButtonText
+              onClick={() => setIsNITR((prev) => !prev)}
+              className='cursor-pointer'
+              outline
+            >
+              Change
+            </ButtonText>
+          </Body2>
+        )}
       </RegistrationCardTitle>
 
-      <InputContainer>
-        {Object.entries(inputData)
-          .filter(([, value]) =>
-            isNITR ? SHOW.NITR.includes(value.show) : SHOW.NON_NITR.includes(value.show),
-          )
-          .map(([key, value], index, array) => {
-            let gridCols;
-            if (array.length <= 4) {
-              gridCols = 'span 4';
-            } else if (index === array.length - 1 && index % 2 === 0) {
-              gridCols = '2 / 4';
-            } else {
-              gridCols = 'span 2';
-            }
+      {isNITR ? (
+        <>
+          <InputContainer>
+            {Object.entries(inputData)
+              .filter(([, value]) =>
+                isNITR ? SHOW.NITR.includes(value.show) : SHOW.NON_NITR.includes(value.show),
+              )
+              .map(([key, value], index, array) => {
+                let gridCols;
+                if (array.length <= 4) {
+                  gridCols = 'span 4';
+                } else if (index === array.length - 1 && index % 2 === 0) {
+                  gridCols = '2 / 4';
+                } else {
+                  gridCols = 'span 2';
+                }
 
-            return (
-              <div key={key} style={{ gridColumn: gridCols }}>
-                <Input
-                  data={value}
-                  key={key}
-                  onChange={(e) => setInputValue(key, e.target.value)}
-                  onBlur={onBlur}
-                />
-              </div>
-            );
-          })}
-      </InputContainer>
+                return (
+                  <div key={key} style={{ gridColumn: gridCols }}>
+                    <Input
+                      data={value}
+                      key={key}
+                      onChange={(e) => setInputValue(key, e.target.value)}
+                      onBlur={onBlur}
+                    />
+                  </div>
+                );
+              })}
+          </InputContainer>
 
-      <Button
-        type='submit'
-        variant='filled'
-        text='Register'
-        width='350px'
-        disabled={!verified && isNITR}
-        tooltip={verified ? '' : 'Verify your credentials'}
-        onClick={registerUser}
-      />
+          <Button
+            type='submit'
+            variant='filled'
+            text='Register'
+            width='350px'
+            disabled={!verified && isNITR}
+            tooltip={verified ? '' : 'Verify your credentials'}
+            onClick={registerUser}
+          />
+        </>
+      ) : (
+        <>
+          <Heading4 font='Inter' style={{ textAlign: 'center', maxWidth: '500px' }}>
+            Registration for non-NITR students has been closed for this year. Please come back next
```

Comment:
Change the text to "Registration for NON-NITR Students has been closed. Please reach out to INNOVISION team in case of any query"